WebApp/calculator/views.py

```diff
@@ -1,6 +1,66 @@
 from django.shortcuts import render
 
-
-# Create your views here.
 def calculator(request):
-    return  render(request, 'calculator/calculator.html')
+    def initContent():
+        return {
+            'displayResult': 0,
+            'lastOperator': '=',
+            'previousValue': 0,
+            'newValue': 0,
+            'isOperatorClicked': False
+        }
+    content = initContent()
+    if request.method == 'POST':
```

Comment:
if you use `if request.method != 'POST':`
you can reduce the indent for following lines